generate-offline-map/README.metadata.json

```diff
@@ -1,35 +1,38 @@
 {
-  "category": "Edit and Manage Data",
-  "description": "Take a web map offline.",
-  "formal_name": "GenerateOfflineMap",
-  "ignore": false,
-  "images": [
-    "generate-offline-map.png"
-  ],
-  "keywords": [
-    "download",
-    "offline",
-    "save",
-    "web map",
-    "GenerateOfflineMapJob",
-    "GenerateOfflineMapParameters",
-    "GenerateOfflineMapResult",
-    "OfflineMapTask",
-    "Portal"
-  ],
-  "language": "kotlin",
-  "redirect_from": [
-    "/android/latest/sample-code/generate-offline-map.htm"
-  ],
-  "relevant_apis": [
-    "GenerateOfflineMapJob",
-    "GenerateOfflineMapParameters",
-    "GenerateOfflineMapResult",
-    "OfflineMapTask",
-    "Portal"
-  ],
-  "snippets": [
-    "src/main/java/com/esri/arcgismaps/sample/generateofflinemap/MainActivity.kt"
-  ],
-  "title": "Generate offline map"
-}
+    "category": "Edit and Manage Data",
+    "description": "Take a web map offline.",
+    "formal_name": "GenerateOfflineMap",
+    "ignore": false,
+    "images": [
+        "generate-offline-map.png"
+    ],
+    "keywords": [
+        "download",
+        "geoviewcompose",
+        "offline",
+        "save",
+        "web map",
+        "GenerateOfflineMapJob",
+        "GenerateOfflineMapParameters",
+        "GenerateOfflineMapResult",
+        "OfflineMapTask",
+        "Portal"
+    ],
+    "language": "kotlin",
+    "redirect_from": [
+        "/android/latest/sample-code/generate-offline-map.htm"
+    ],
```

Comment:
We shouldn't add redirects from Runtime to Maps SDK. Redirects would be helpful when the currently published sample would be renamed to a different name, so we should redirect from old sample name. Not needed here.